wazo_call_logd/plugins/cdr/http.py

```diff
@@ -182,6 +187,43 @@ def visible_tenants(self, recurse=True):
         else:
             return [tenant_uuid]
 
+
+class RecordingsMediaResource(RecordingMediaAuthResource):
+    @required_acl(
+        'call-logd.cdr.recordings.media.delete',
+        extract_token_id=extract_token_id_from_query_or_header,
+    )
+    def delete(self):
+        args = RecordingMediaDeleteRequestSchema().load(request.get_json(force=True))
+        tenant_uuids = self.visible_tenants(True)
+        call_log_ids = args['cdr_ids']
+        recordings_to_delete = []
+        # We do not want to delete any recording if one of the CDR has not been found
+        for cdr_id in call_log_ids:
+            cdr = self.service.find_cdr(cdr_id, tenant_uuids)
+            if not cdr:
+                raise CDRNotFoundException(details={'cdr_id': cdr_id})
+            recordings_to_delete.extend(cdr.recordings)
+
+        for recording in recordings_to_delete:
+            try:
+                self.service.delete_media(
+                    recording.call_log_id, recording.uuid, recording.path
+                )
+            except PermissionError:
+                logger.error('Permission denied: "%s"', recording.path)
+                raise RecordingMediaFSPermissionException(
+                    recording.uuid, recording.path
+                )
+            except FileNotFoundError:
+                logger.info(
+                    'Recording file already deleted: "%s". Marking as such.',
+                    recording.path,
+                )
```

Comment:
Maybe we should use another error to be able to return cdr_id. Because the user of rest api will not know from which cdr_id come from the error